GUI/window.py

```diff
@@ -213,23 +209,21 @@ def __init__(self, **kwargs):
                                         # size_hint=(1, 0.17),
                                         pos_hint={'x': dp(-0.025), 'y': dp(0.35)})
 
-        self.light.change_color(self.micro_sim.traffic_lights_binary())
-        self.seven_segment_display.activate_segments(
-            self.micro_sim.seven_segment_binary())
-
         # Create variable of scheduling instance so that it can be turned on and off,
         # to avoid repeat of the same thread
         self.event_on = Clock.schedule_interval(
             self.light.intermittent_off, 0.5)
         self.event_off = Clock.schedule_interval(
             self.light.intermittent_on, 0.3)
 
+        # Creates a clock thread that updates all tables and i/o's every 2 seconds. Does not get cancelled.
```

Comment:
# Creates a clock thread that updates all tables and i/o's every 0.2 seconds. Does not get cancelled.

Reply:
Done